smt/surrogate_models/tests/test_kpls.py

```diff
@@ -46,25 +46,6 @@ def test_predict_output(self):
             kriging.set_training_values(x, y)
             self.assertRaises(ValueError, lambda: kriging.train())
 
-    def test_kpls_training_with_zeroed_outputs(self):
```

Comment:
Why removing this test?

Reply:
It was a test for a regression in scikit-learn 0.24 on a ill-formed case which now just generates a warning, as KPLS is tested elsewhere, I just removed the test. I can bring it back if you a find a value in it.